src/main/java/de/fraunhofer/aisec/cpg/graph/declarations/RecordDeclaration.java

```diff
@@ -64,6 +65,10 @@
   @SubGraph("AST")
   private List<PropertyEdge<RecordDeclaration>> records = new ArrayList<>();
 
+  @Relationship(value = "PARAMETERS", direction = "OUTGOING")
+  @SubGraph("AST")
+  private List<PropertyEdge<ParameterizedType>> parameters = new ArrayList<>();
```

Comment:
The name parameters is misleading. I would recommend something like typeParameters

Reply:
I have removed the typeParameters edges, since they will be modelled differently, due to the implementation of CPP Templates. They are now stored in the TypeManager.